Makefile

```diff
@@ -53,10 +51,23 @@ ifneq ($(LLD),0)
 endif
 
 UNAME := $(shell uname)
+UNAMEM := $(shell uname -m)
 ifneq ($(UNAME), Darwin)
-    LDFLAGS += -Wl,-export-dynamic -lstdc++fs
+  LDFLAGS += -Wl,-export-dynamic -lstdc++fs
+  EXPORTERS := -Wl,--whole-archive ExporterTest/ExporterTest.a -Wl,--no-whole-archive
+else
+  EXPORTERS := -Wl,-force_load ExporterTest/ExporterTest.a
+  ifeq ($(UNAMEM),arm64)
+    ifeq ($(shell brew list libpng > /dev/null 2>&1; echo $$?),0)
+      LDFLAGS += -L $(shell brew --prefix)/lib
+      INC += -I $(shell brew --prefix)/include
+      else
+      $(error Please install libpng via Homebrew)
```

Comment:
You could add instructions in the readme on how to install libpng using Homebrew

Reply:
Oh yeah, good point. Done.